File: C++ Dasar/Contoh Statement/Contoh Switch Case/Switch Case.cpp
#include <iostream>

using namespace std;

int main() {
	
int pilihan;
    
    cout << "Menu :\n" << endl;
    cout << "1. Contoh Ke Satu           " << endl;
    cout << "2. Contoh Ke Dua            " << endl;
    cout << "3. Contoh Ke Tiga           " << endl;
    cout << "4. Contoh Ke Empat	         \n" << endl;
    cout << "Masukan Pilihan : "; 
    cin >> pilihan;
    switch (pilihan)
    {
        case 1: {
			cout << "Contoh Ke-1 Berhasil Dicoba" << endl;
       	    break;
    	}
		case 2: {
            cout << "Contoh Ke-2 Berhasil Dicoba" << endl;
            break;
       	}
      	case 3: {
        	cout << "Contoh Ke-3 Berhasil Dicoba" << endl;
            break;
        }
        case 4: {
        	cout << "Contoh Ke-4 Berhasil Dicoba" << endl;
        	break;
        }
    	default: {
            cout << "Masukkan Pilihan Dengan Benar" << endl;
            break;
        }
    }
 
 	return 0;
 
}

File: C# Dasar/Contoh Statement/Contoh Switch Case/Switch Case.cs
﻿using System;

class main
{
    static void Main(string[] args)
    {
        int pilihan;

        Console.WriteLine("Menu :\n");
        Console.WriteLine("1. Contoh Ke Satu");
        Console.WriteLine("2. Contoh Ke Dua");
        Console.WriteLine("3. Contoh Ke Tiga");
        Console.WriteLine("4. Contoh Ke Empat");
        Console.Write("Masukan Pilihan : ");
        pilihan = int.Parse(Console.ReadLine());

        switch (pilihan)
        {
            case 1:
                Console.WriteLine("Contoh Ke-1 Berhasil Dicoba");
                break;
            case 2:
                Console.WriteLine("Contoh Ke-2 Berhasil Dicoba");
                break;
            case 3:
                Console.WriteLine("Contoh Ke-3 Berhasil Dicoba");
                break;
            case 4:
                Console.WriteLine("Contoh Ke-4 Berhasil Dicoba");
                break;
            default:
                Console.WriteLine("Masukkan Pilihan Dengan Benar");
                break;
        }

    }
}